dockerfiles/init/modules/lighttpd/templates/lighttpd.conf.erb

```diff
@@ -0,0 +1,323 @@
+###############################################################################
+# Default lighttpd.conf for Gentoo.
```

Comment:
this is default content of lighttpd.conf it have info for all operations systems I think